packages/custom-functions-metadata/src/parseTree.ts

```diff
@@ -138,6 +138,27 @@ const TYPE_CUSTOM_FUNCTION_CANCELABLE = {
 };
 const TYPE_CUSTOM_FUNCTION_INVOCATION = "customfunctions.invocation";
 
+// These does not work if the developer is using namespace/type alias. To support that, we'd need to
+// use typeChecker.getFullyQualifiedName, which in turns requires creating a program out of the source file
+// (which we don't do now) and passing the type checker all the way down to getParamType. This would be a
+// larger refactor (e.g., rewriting parseTree into a class) and is tracked as a work item.
+const CELLVALUETYPE_MAPPINGS = {
+  "Excel.CellValue": "cellvalue",
+  "Excel.BooleanCellValue": "booleancellvalue",
+  "Excel.DoubleCellValue": "doublecellvalue",
+  "Excel.EntityCellValue": "entitycellvalue",
+  "Excel.ErrorCellValue": "errorcellvalue",
```

Comment:
Aren't there other error cell value types? They should all be supported.

It would be nice to have type info which could tell that the type is derived from Excel.CellValue, but currently there is no type info for these types with custom-functions-metadate. If the types are pulled out into an excel.cellvalue.d.ts for the type info, then that could be used by the code here and perhaps have better knowledge about the type hierarchy, or be written where new types could be added and the checks in here can just tell whether the class derives from Excel.CellValue. But this approach based on the list of type names is OK for now.

Reply:
We don't support error subtypes now.